Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1118,6 +1118,17 @@ def resolve_ip_address(ip):
     return None
 
 
+def convert_url_to_ascii_character(url_name):
+    regex_groups = re.search('([a-zA-Z\W]*)([^a-zA-Z\W]*)([a-zA-Z\W]*)', url_name)
```

Comment:
what if there is more than one special character (for example, `www.MüMünchen.com`)?